bolts/models/laftr.py

```diff
@@ -129,29 +133,27 @@ def _inference_step(self, batch: DataBatch, stage: str) -> Dict[str, Tensor]:
 
     def _loss_adv(self, s_pred: Tensor, batch: DataBatch) -> Tensor:
         # For Demographic Parity, for EqOpp is a different loss term.
-        s_target = batch.s.view(-1, 1)
-        y_target = batch.y.view(-1, 1)
         if self.fairness is FairnessType.DP:
-            s0 = self._adv_clf_loss(s_pred[batch.s == 0], s_target[batch.s == 0])
-            s1 = self._adv_clf_loss(s_pred[batch.s == 1], s_target[batch.s == 1])
+            s0 = self._adv_clf_loss(s_pred[batch.s == 0], batch.s[batch.s == 0])
+            s1 = self._adv_clf_loss(s_pred[batch.s == 1], batch.s[batch.s == 1])
             loss = (s0 + s1) / 2
```

Comment:
I feel like this could be a 1 liner

Reply:
@MylesBartlett any ideas?

Reply:
err just do `loss(s_pred, s)?`

Reply:
no, because I need the average of each group, which is different

Reply:
you could do it by reshaping and reducing